marathon_lb.py

```diff
@@ -1729,6 +1738,19 @@ def load_json(data_str):
     # Process arguments
     arg_parser = get_arg_parser()
     args = arg_parser.parse_args()
+    if args.sentry:
+        import sentry
+        from raven import Client
+        global sentry_user
+        global sentry_pwd
+        global sentry_endpoint
+        global client
```

Comment:
only `client` really needs to be a global, and it should be initialized to `None` somewhere.